### PR TITLE
Col - fix disabled width props from being passed to div

### PR DESCRIPTION
**Pull request by hosmelq**
Sometimes you don't want the `col` class by default, and setting xs={false} works but i got this error

`
Unknown prop 'xs' on <div> tag. Remove this prop from the element.
`

This prop is not removed from attributes.

https://github.com/reactstrap/reactstrap/blob/master/src/Col.js#L63

### CODE REVIEW

File: src/Col.js [hunk 1]
```diff
@@ -28,10 +28,6 @@ const propTypes = {
   cssModule: PropTypes.object,
 };
 
-const defaultProps = {
```
Comment (TheSharpieOne) [inline src/Col.js]: `defaultProps` is used below

Comment (hosmelq) [inline src/Col.js]: My bad, thanks.

File: src/Col.js [hunk 2]
```diff
@@ -78,6 +78,10 @@ const Col = (props) => {
     }
   });
 
+  if (attributes['xs']) {
```
Comment (eddywashere) [inline src/Col.js]: I'd be happy to add back support for `false` as I think `Col` should have some sort of default that takes care of the typical use case. I recently and foolishly moved `delete attributes[colWidth];` after the prop check, it should instead revert back to the following in `colWidths.forEach(colWidth => {`:

```
// always clear out width props from attributes
delete attributes[colWidth];

if (!columnProp) {
  return;
}
```

Comment (hosmelq) [inline src/Col.js]: Do you already moved this ?

Comment (eddywashere) [inline src/Col.js]: It was previously before the check, (see commit](https://github.com/reactstrap/reactstrap/pull/267/files#diff-771c8561c9896cbffb7d4b7f17506333)

image

File: src/Col.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ const propTypes = {
 };
 
 const defaultProps = {
-  xs: true
+    xs: true
```
Comment (eddywashere) [inline src/Col.js]: nit: can you remove the extra spaces?

**Pull request merged**